mediapipe/calculators/geti/scripts/setup_opencv.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+export OPENCV_VERSION=4.3.0
```

Comment:
Why using old version?

Reply:
Mediapipe officially supports up to 4.1. (https://developers.google.com/mediapipe/framework/getting_started/install)

In a concession that was moved up to 4.3.

Reply:
Removing scripts.